surprise_decorator.py

```diff
@@ -0,0 +1,18 @@
+def change_original_functionality_decorator(func: callable):
+    """
+    A decorator that will make the function print “surprise!” instead of its original functionality.
+    :param func:The original function.
+    :return:The function print_surprise.
+    """
+    def print_surprise():
+        print("surprise!")
+    return print_surprise
+
+
+@change_original_functionality_decorator
+def print_name():
+    print("print name")
+
+
+if __name__ == '__main__':
+    print_name()
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add